extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -61,7 +61,7 @@ amp-sidebar::part(sidebar) {
  * styling is not shown directly on amp-sidebar element.
  */
 amp-sidebar {
-  visibility: hidden !important;
+  /*visibility: hidden !important;*/
```

Comment:
Needed for current iteration or else `amp-sidebar` is always invisible.

Reply:
Do we require `layout=nodisplay` for a sidebar? If not, we can also always make it `display:none` by default until it's open. E.g.

```
amp-sidebar:not([open]) {
  display: none;
}
```